transport_plugins/jlink/iotile_transport_jlink/jlink.py

```diff
@@ -299,6 +299,17 @@ def _open_script_interface(self, conn_id, callback):
 
         callback(conn_id, self.id, True, None)
 
+    def _open_streaming_interface(self, conn_id, callback):
```

Comment:
Is this required to add now?  The JLink fw and python package does not support streaming so trying to open the streaming interface should raise a `StreamOperationNotSupported` error which should happen already just by not implementing this function.

Adding this function will make that error go away but streaming will still fail since the underlying channel doesn't support it yet.

I would recommend removing this function until we actually add jlink support for streaming.  It will mean that a call to `enable_streaming` in HardwareManager will raise an exception but I would argue that is correct since streaming is not supported.

Reply:
Ok.. I added this function before discussing the issues with streaming... I'll remove it.